### PR TITLE
Morefields

### PR DESCRIPTION
**Pull request by gabzn**

### CODE REVIEW

File: client/src/components/Dashboard.js [hunk 1]
```diff
@@ -48,6 +48,8 @@ function Dashboard() {
 	const [step, setStep] = useState(1);
 	const [storeName, setStoreName] = useState('');
 	const [billAmount, setBillAmount] = useState('');
+	const [tips, setTips] = useState('');
+	const [tax, setTax] = useState('');
```
Comment (gabzn) [inline client/src/components/Dashboard.js:52]: Added both needed fields.

**Review by WenxuC — APPROVED** [review]

File: client/src/components/Dashboard.js [hunk 2]
```diff
@@ -104,6 +106,8 @@ function Dashboard() {
 		const body = {
 			storeName: storeName,
 			amount: billAmount,
+			tax: tax,
+			tips: tips,
```
Comment (gabzn) [inline client/src/components/Dashboard.js:110]: put them both in the body.

File: client/src/components/Dashboard.js [hunk 3]
```diff
@@ -155,13 +159,14 @@ function Dashboard() {
 		setStep(1);
 		setStoreName('');
 		setBillAmount('');
+		setTips('');
+		setTax('');
```
Comment (gabzn) [inline client/src/components/Dashboard.js:163]: Added them into the resetForm function.

File: server/routes/dashboardRoute.js [hunk 4]
```diff
@@ -122,8 +122,7 @@ router.post('/email', auth, async (req, res) => {
 			});
 		}
 		
-	
-		res.send(`Email sented`);
+		res.send(`Email sent`);
```
Comment (gabzn) [inline server/routes/dashboardRoute.js:125]: Fixed grammar error.

**Pull request merged**